### PR TITLE
Some editorial improvement in documents

### PR DESCRIPTION
**Pull request by xiaoyinl**
[NOTES.WIN]: this -> these
[doc/HOWTO/certificates.txt]: "If not, There are" -> "If not, there are"
[doc/HOWTO/proxy_certificates.txt]: use HTTPS for link to RFC 3820
<del>[doc/man7/Ed25519.pod]: add link to rfc8032</del>

### CODE REVIEW

File: doc/man7/Ed25519.pod [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (InfoHunter) [inline doc/man7/Ed25519.pod]: This new line should not be inserted, and this also broke 'doc-nit' check in travis-ci.

Comment (xiaoyinl) [inline doc/man7/Ed25519.pod]: @InfoHunter Fixed. Thank you!

File: doc/man7/Ed25519.pod [hunk 2]
```diff
@@ -7,9 +7,9 @@ Ed25519 - EVP_PKEY Ed25519 support
 =head1 DESCRIPTION
 
 The B<Ed25519> EVP_PKEY implementation supports key generation, one shot
-digest sign and digest verify using PureEdDSA and B<Ed25519> (see RFC8032).
-It has associated private and public key formats compatible with
-draft-ietf-curdle-pkix-04.
+digest sign and digest verify using PureEdDSA and B<Ed25519> (see
+L<RFC8032|https://tools.ietf.org/html/rfc8032>). It has associated private
```
Comment (richsalz) [inline doc/man7/Ed25519.pod]: There are about 150 RFC references in our manpages, and changing just this one to be an Link seems wrong.  Please turn edit to "RFC 8032"  Or fix all of the others :)

Comment (xiaoyinl) [inline doc/man7/Ed25519.pod]: @richsalz I have removed this link. Thank you for your review!

**Comment by xiaoyinl** [comment]
Do you want me to squash the commits before you merge?

**Comment by richsalz** [comment]
squashed and merged to master, thank you!

**Pull request closed without merging**